include/SFML/Graphics/View.hpp

```diff
@@ -109,7 +109,7 @@ class SFML_GRAPHICS_API View
     /// view are displayed, expressed as a factor (between 0 and 1)
     /// of the size of the RenderTarget to which the view is applied.
     /// For example, a view which takes the left side of the target would
-    /// be defined with View.setViewport(sf::FloatRect(0, 0, 0.5, 1)).
+    /// be defined with View.setViewport(sf::FloatRect({ 0.f, 0.f }, { 1.f, 1.f })).
```

Comment:
This does not correspond to the rest of the sentence.
```suggestion
    /// be defined with View.setViewport(sf::FloatRect({ 0.f, 0.f }, { 0.5f, 1.f })).
```

Reply:
Also, we should probably use a SFML code style, i.e. lowercase variable names and no spaces (IIRC):
```cpp
view.setViewport(sf::FloatRect({0.f, 0.f}, {0.5f, 1.f}))
```